docs/sources/administration/configuration.md

```diff
@@ -908,6 +910,34 @@ Syslog tag. By default, the process's `argv[0]` is used.
 
 <hr>
 
+## [log.frontend]
+
```

Comment:
Can we add the following information in the form of a table for better flow/structure? Thanks.

Reply:
Thank you for the review @achatterjee-grafana! 
I tried to be consistent with the formatting of all other content in this file. If we were to use a table, it would make sense to reformat the entire document rather than just this particular section. I think that's out of scope for current PR and would merit it's own issue & PR. Is that ok?

Reply:
@domas, yes I agree. It is a rather long topic and so yes, a general restructuring at a later date is a good idea.